cpp/order/LocalWl.cc

```diff
@@ -34,13 +34,13 @@ void LocalWl::computeYlm(const float theta, const float phi, std::vector<std::co
     for (typename fsph::PointSPHEvaluator<float>::iterator iter(sph_eval.begin_l(m_l, 0, false));
          iter != sph_eval.end(); ++iter)
     {
-        Ylm[(j + m_l) % (2 * m_l + 1)] = *iter;
+        // Manually add the Condon-Shortley phase, (-1)^m, to positive odd m
+        float phase = (j % 2 == 1) ? -1 : 1;
```

Comment:
Given our discussions a few weeks ago, it's probably worth adding a note that `(j % 2) == -1` if j is a _negative_ odd number since that's not the behavior of Python, only C(++).

Reply:
`j` is nonnegative -- it's an unsigned int. We use j to index over m=0, 1, 2, 3, and we compute the negative m values by conjugating the positive m values below. (We only apply the Condon-Shortley phase to positive odd m.)

Reply:
My mistake, I think I got confused between this and Steinhardt with respect to whether the iterator was including the negative `m` values. I think the comment is a bit confusing since you add the Condon-Shortley phase to all positive `m`, it's just that the phase is positive for even `m`, but that's a minor nitpick.